src/pipeline/activator/aviris-l2/requirements-conda.txt

```diff
@@ -0,0 +1,5 @@
+awscli
```

Comment:
We're installing the AWS CLI but this seems more like a development dependency? In other projects, we've used the Terraform container for access to the CLI.

Reply:
Removed from the dev container. For now the subprojects that require it, like `catalogs-aviris`, have it installed for the operations they require.